test/runtests.jl

```diff
@@ -56,6 +72,15 @@ mktempdir() do tmpdir; mktempdir() do depot
         withenv("JULIA_LOAD_PATH" => tmpdir) do # Should work even though Pkg is not in LOAD_PATH
             @test success(`$(test_cmd) --update st -m`)
         end
+        # Test --julia flag
+        if Sys.islinux() && get(ENV, "TRAVIS", nothing) == "true"
```

Comment:
```suggestion
        if Sys.islinux() && get(ENV, "CI", nothing) == "true"
```

All CI's define this